src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx

```diff
@@ -0,0 +1,192 @@
+
+<img
+    title="APM summary page for checkout service"
+    alt="A screenshot showing the APM summary page"
+    src={apmCheckoutServiceinApm}
+/>
+</Step>
+
+<Step>
+
+### Select a time period [#time-picker]
+
+It's important to make sure that you are looking at the right time segment. Based on the message you got from your leadership team, you know the checkout issues are happening now and so you want to hone in on what's actively happening in your system. Click on the **Time Picker** in the top-left, and select **Since 30 minutes ago**.
+
+    <img
+    title="Time picker in APM"
+    alt="A screenshot that demonstrates how to select the time period for APM"
+    src={apmTimePickerinApm}
+/>
+</Step>
+
+<Step>
+
+### Find the problematic transaction [#transactions]
+
+You know that checkout is failing in your app but you're unsure of what's causing the actual error. In order to figure out what part of the checkout process is breaking you need to investigate your apps `Transactions`. A transaction represents the “work” that happens from when your application receives a request to when it sends a response. In order to determine which transaction is breaking, click on the **Transactions** tab.
+
+A slow transaction can be a strong indicator that something is behaving abnormally, so digging deeper into your slowest transaction is a good place to start. Once you click into **Transactions** you will see a list of all the "work" that is happening behind the scenes for your `checkout-service` application. On the left-hand side you will see which transaction is taking the most time. For the purpose of this example, the slowest transaction is `Sinatra/Rubytron/POST #purchaseCart.` Click on the transaction.
+
+<img
+    title="transactions tab in APM summary "
+    alt="A gif showing you how to dive deep into transactions"
+    src={apmTransactionsTabinApmOverview}
+/>
+
+</Step>
+
+<Step>
+### Dig deeper into your slowest transaction [#slow-transactions]
+
+Each transaction is comprised of many different units of "work." The **Transactions detail** page provides a holistic view of a single transaction by breaking it down into individual pieces of "work". The histogram on top visually demonstrates which pieces of the transaction are responding slower than others. 
+
+For the `Sinatra/Rubytron/POST #purchaseCart` transaction there are many different areas where the transaction could be failing: databases, servers, controllers, middleware and more. You can toggle between each different segment of the transaction to identify trends and spikes.
+
+In this case, the controller, `Sinatra/Rubytron/POST` is the slowest segment and is taking 59.83% of the transactions total time. 
+
+    <img
+    title="Slowest transaction"
+    alt="A gif that demonstrates how to select the slowest transaction"
+    src={accountsSelectSlowestTransaction}
+/>
+
+</Step>
+
+<Step>
+### Take a look at throughput 
```

Comment:
Something to think about; it was a little unclear sometimes how the gifs or images related to the text around it. You might peruse the built doc and see which ones stand out as being like that, and then either add in introductory lead in (This gif shows...) or do a diagram.

Reply:
good idea! thanks!